androidNode/src/androidMain/kotlin/network/bisq/mobile/android/node/MobileNetworkServiceConfig.kt

```
@@ -0,0 +1,321 @@
+                            // Complex nested format: [{ external: [...] }]
+                            try {
+                                val firstEntryObj = firstEntry as com.typesafe.config.ConfigObject
+                                val clearConfig = firstEntryObj.toConfig()
+                                if (clearConfig.hasPath("external")) {
+                                    clearConfig.getStringList("external").stream()
+                                        .map { Address.fromFullAddress(it) }
+                                        .collect(Collectors.toSet())
+                                } else {
+                                    emptySet()
+                                }
+                            } catch (e: ClassCastException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)